tests/unit_tests/test_clustering.py

```diff
@@ -38,6 +38,20 @@ def test_data() -> Dict:
     """Test data for TestClustering."""
     with open(os.path.join("test_data", "yolo_data.json")) as json_file:
         data: Dict = list(json.load(json_file).values())[0]
+
+    # Make category to id mapping
+    category_to_id = {i: bb.split()[0] for i, bb in enumerate(data)}
+    # Go through bounding boxes and change class to an integer while creating a mapping
+    sheet_data = [
+        f"{list(category_to_id.keys())[list(category_to_id.values()).index(bb.split()[0])]} {' '.join(bb.split()[1:])}"
```

Comment:
Well that's... obtuse.

Reply:
Yeah I'm a bit confused why we have to create these maps for converting yolo data to bounding box objects haha. Let me know if you have any suggestions.